src/main/java/software/amazon/awssdk/http/apache/internal/conn/SdkTlsSocketFactory.java

```diff
@@ -5,9 +5,17 @@
 
 package software.amazon.awssdk.http.apache.internal.conn;
 
+#if ANDROID
+//  Apache httpcomponents {core-4.4.1x,httpclient-4.5.13}  does not compatible with Android, we should rename classes to avoid conflict with legacy implementation used by Android
+import org.gapache.http.HttpHost;
+import org.gapache.http.conn.ssl.SSLConnectionSocketFactory;
+import org.gapache.http.protocol.HttpContext;
+#else
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error on tokens, delete these tokens

This file contains syntax errors and can&#39;t be analyzed by all of CodeGuru Reviewer&#39;s detectors. Fix the syntax errors and run the repository analysis or submit the pull request again.